### PR TITLE
Separating metal-lb ip-address pool definition

### PR DESCRIPTION
**Pull request by arora-sagar**
- This is needed for the oai exercise document.

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes # Currently in the exercise document for OAI, there is no method to define metal-lb ip-address pool for workload clusters. In R2 even workload clusters use metal lb as NRF exposes its service via Metal LB for UPF to register. 

**Special notes for your reviewer**:

Breaking 001-infra.sh in two parts was a easier choice than renaming all the scripts. 

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: e2e/tests/oai/001-infra.sh [hunk 1]
```diff
@@ -10,7 +10,7 @@
 ##############################################################################
 
 ## TEST METADATA
-## TEST-NAME: Deploy and interconnect OAI clusters
+## TEST-NAME: Deploy and interconnect OAI clusters Part I
```
Comment (electrocucaracha) [inline e2e/tests/oai/001-infra.sh:13]:
```suggestion
## TEST-NAME: Deploy and interconnect OAI clusters
```

File: e2e/tests/oai/001-infra-metal-lb.sh [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+##############################################################################
+# Copyright (c) 2023 The Nephio Authors.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Apache License, Version 2.0
+# which accompanies this distribution, and is available at
+# http://www.apache.org/licenses/LICENSE-2.0
+##############################################################################
+
+## TEST METADATA
+## TEST-NAME: Deploy and interconnect OAI clusters Part II
```
Comment (electrocucaracha) [inline e2e/tests/oai/001-infra-metal-lb.sh:13]:
```suggestion
## TEST-NAME: Define IP Address pools on OAI clusters
```

**Comment by electrocucaracha** [comment]
@arora-sagar I [think](http://prow.nephio.io/view/gs/prow-nephio-sig-release/pr-logs/pull/nephio-project_test-infra/235/e2e-oai-ubuntu-jammy/1753203394358546432#1:build-log.txt%3A50892) that you have to rename the 001 bash scripts differently. Like `001b-infra-metal-lb.sh` and `001a-infra.sh`

**Comment by arora-sagar** [comment]
@electrocucaracha I don't understand the reason now for the failure. It failed once that is okay but now I tested manually the scripts and I don't get the error because of the networking script. Earlier I had cluster section missing 

```
declare -A clusters
clusters=(
    ["core"]="172.18.16.0/20"
    ["regional"]="172.18.32.0/20"
    ["edge"]="172.18.48.0/20"
)
```

But when I added it the run still showed me the same error as the previous commit.  Can you help me understand this?

**Pull request merged**